core/src/test/java/org/elasticsearch/indices/recovery/RecoveryStateTest.java

```diff
@@ -154,7 +154,7 @@ public void run() {
         if (randomBoolean()) {
             timer.stop();
             assertThat(timer.stopTime(), greaterThanOrEqualTo(timer.startTime()));
-            assertThat(timer.time(), equalTo(timer.stopTime() - timer.startTime()));
+            assertThat(timer.time(), lessThanOrEqualTo(timer.stopTime() - timer.startTime()));
```

Comment:
Are we sure we can use lessThanOrEqualTo here? we can also just assert that time() > 0 imho.